app/models/dttp/degree_qualification.rb

```diff
@@ -4,10 +4,28 @@ module Dttp
   class DegreeQualification < ApplicationRecord
     self.table_name = "dttp_degree_qualifications"
 
+    belongs_to :dttp_trainee,
+               foreign_key: :contact_dttp_id,
+               primary_key: :dttp_id,
+               optional: true,
+               inverse_of: :degree_qualifications,
+               class_name: "Dttp::Trainee"
+
     validates :response, presence: true
 
     enum state: {
       unprocessed: 0,
+      processed: 1,
```

Comment:
We should get together and have a think about what the states are. `processed` could be successfully or not so we probably shouldn't use that.

Reply:
Fixed in https://github.com/DFE-Digital/register-trainee-teachers/pull/1787/commits/b0eb4573edd96a9a0864c69ba8691baad34b05b3